packages/sentinel_one_cloud_funnel/data_stream/event/elasticsearch/ingest_pipeline/default.yml

```
@@ -1976,10 +2020,23 @@ processors:
       field: json.src.process.publisher
       target_field: sentinel_one_cloud_funnel.event.src.process.publisher
       ignore_missing: true
+  - set:
+      field: process.code_signature.subject_name
+      copy_from: sentinel_one_cloud_funnel.event.src.process.publisher
+      ignore_empty_value: true
   - rename:
       field: json.src.process.verifiedStatus
       target_field: sentinel_one_cloud_funnel.event.src.process.verified_status
       ignore_missing: true
+  - set:
+      field: process.code_signature.trusted
+      value: true
+      if: ctx.sentinel_one_cloud_funnel?.event?.src?.process?.verified_status?.contains('verified') == true
```

```suggestion
      if: ctx.sentinel_one_cloud_funnel?.event?.src?.process?.verified_status?.contains('verified')
```
Not sure if `== true` is required

---

Ah okay 👍🏼